.travis.yml

```diff
@@ -23,11 +23,10 @@ jobs:
 
 cache:
   directories:
-  - $HOME/.sbt/1.0/dependency
-  - $HOME/.sbt/boot/scala*
-  - $HOME/.sbt/launchers
+  - $HOME/.sbt/zinc
   - $HOME/.ivy2/cache
   - $HOME/.nvm
+  - $HOME/.coursier
```

Comment:
Should be `$HOME/.cache/coursier`

Reply:
I need to document that too.

Reply:
Ok, fixed it myself.

Reply:
Looks like I need to update all of my .travis.yml!

Reply:
It depends… if you're sure `~/.coursier` exists when the job starts, it will get picked. Else, it's `~/.cache/coursier`. Caching `~/.cache/coursier` everywhere is safer.